libs/langchain/langchain/document_loaders/parsers/pdf.py

```diff
@@ -149,3 +149,79 @@ def lazy_parse(self, blob: Blob) -> Iterator[Document]:
                 )
                 for page in doc.pages
             ]
+
+
+class AmazonTextractPDFParser(BaseBlobParser):
+    """Sends PDF files to Amazon Textract and parses them to generate Documents.
+    For parsing multi-page PDFs, they have to reside on S3.
+    """
+
+    try:
+        import textractcaller as tc
```

Comment:
This import is being evaluated at module import time since it's in the global namespace of the class. We typically nest these imports into the methods where they are used to get around that issue.

I believe the reason this is here is to get the type checking for textract_features in the init below.